services/docker/generate.go

```diff
@@ -105,12 +105,15 @@ func UnmarshalledDefaults() conf.ComposeDotTok {
 			log.Fatalf("Error setting Compose file defaults: %v", err)
 		}
 
-		err = yaml.Unmarshal(dockertmpl.TokaidoDockerSiteVolumeAttach(conf.GetConfig().Tokaido.Project.Path), &tokStruct)
+		err = yaml.Unmarshal(dockertmpl.TokaidoDockerSiteVolumeAttach(conf.GetProjectPath()), &tokStruct)
 		if err != nil {
 			log.Fatalf("Error attaching persistent /tokaido/site volume: %v", err)
 		}
 
 	} else {
+		fmt.Println("CONF AT THIS STAGE::::\n")
+		fmt.Println(conf.GetConfig())
+		fmt.Println("\n\n\n\n")
```

Comment:
Looks like debug messages? Did you want to leave this in?

Reply:
ooops!